backend/src/routes/ordersRoutes.js

```diff
@@ -0,0 +1,51 @@
+const express = require('express');
+const router = express.Router();
+
+//LISTAR TODOS (COM VERIFICAÇÃO SE FOI DELETADO)
+router.get('/list', (req, res) => {
+    let query = "SELECT * FROM orders";
+    if (req.query.showDeleted != "1") {
+        query += ' WHERE deleted_at IS NULL';
+    }
+    req.connect.query(query, (error, result) => {
+        if (error) {
+            res.status(404).send(error);
+        } else {
+            res.send(result);
+        }
+    });
+});
+//LISTAR POR ID (COM VERIFICAÇÃO SE FOI DELETADO)
+router.get('/list/:id', (req, res) => {
+    let query = `SELECT * FROM orders WHERE id=${req.params.id}`;
+    if (req.query.showDeleted != "1") {
+        query += ' AND deleted_at IS NULL';
+    }
+    req.connect.query(query, (error, result) => {
+        if (error) {
+            res.status(404).send(error);
+        } else {
+            res.send(result);
+        }
+    });
+});
+
+router.post('/post', (req, res) => {
+})
+router.put('/put/:id', (req, res) => {
+})
+router.delete('/delete/:id', (req, res) => {
+})
+
+//ROUTER PARA ADICIONAR ENCOMENDA (AddOrder)
+
+//ROUTER PARA LISTAR ENCOMENDAS  (listOrders) 
+
+//ROUTER PARA PESQUISAR ENCOMENDAS (searchOrders)
```

Comment:
As rotas de consulta(Pesquisa) não deveriam estar aqui? Pelo que vi, só tem as rotas de listagem das encomendas.

Reply:
não pois eu faço o filtro de pesquisa com base na listagem.

Reply:
Isso eu entendo. To me referindo ao Pull Request "Adicionando rotas de Consultas", Porém, só tem as rotas de listagem. Não deveria ter as rotas de Consultas(Pesquisa) de fato no código para ser coerente ao Pull Request?

Reply:
Como falei as consultas serão através da página de listagem, até o presente momento. Então me diga qual a diferença entre o título consulta ou listagem ? Se fiz a listagem e já falei que a consulta será feita na tela de listagem ? Mais se desejar renomeio o pull pra ficar feliz.